androidApp/src/main/java/dev/johnoreilly/confetti/utils/LocalDateTime.kt

```diff
@@ -0,0 +1,9 @@
+package dev.johnoreilly.confetti.utils
+
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.toJavaLocalDate
+import java.time.format.DateTimeFormatter
+
+/** @see [DateTimeFormatter.format] */
+fun DateTimeFormatter.format(date: LocalDate): String =
```

Comment:
I have tried to place it inside `shared` but I couldn't import it on the android module. I will place it as Android-only for now.

Reply:
@marcellogalhardo I'll merge this and follow up then and see if/where date stuff might need to move to

Reply:
Sounds good to me. Please see [here](https://github.com/joreilly/Confetti/pull/359#discussion_r1141455461).